### DISCUSSION TITLE
URL Shortner as a Service like Dub

### DISCUSSION THREAD

**Discussion by deadcoder0904** [coollabsio/coolify#1388] [Service Template Requests]
https://dub.co (self-host fork -> https://github.com/Snazzah/stub) & many other url shortners like https://github.com/thedevs-network/kutt or https://github.com/NayamAmarshe/MagLit or https://github.com/boojack/slash or https://rdt.li/

i like magiclit or stub to be self-hosted 1-click like digitalocean marketplace.

**Comment by kesylo** [comment]
I really want to use dub. please can you integrate this? https://dub.co/

**Comment by wottpal** [comment]
Having dub.co as an option would be dope! 🔥

**Comment by jooles** [comment]
would love this as well, fingers crossed.. hope that @peaklabs-dev sees this, he's unbelievably fast.. 😅

**Reply by peaklabs-dev** [reply]
Unfortunately, stub is not ready for production yet, and dub still uses too many external dependencies for it to make sense, see my issue here https://github.com/dubinc/dub/issues/598. You can use [Shlink](https://shlink.io/?utm_source=coolify.io) for now, as it is available as a one-click template. But as soon as dub fixes the linked issue, I will happily add this template.
